.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file and add it to ignore list.

Reply:
Redundant file has been removed 7ebce11 .